physics/kepler_orbit.hpp

```diff
@@ -32,6 +32,10 @@ class KeplerOrbit {
               Body const& secondary,
               KeplerianElements<Frame> const& elements_at_epoch,
               Instant const& epoch);
+  KeplerOrbit(MassiveBody const& primary,
+              Body const& secondary,
+              RelativeDegreesOfFreedom<Frame> const& state_vectors,
```

Comment:
I'm not fond of `primary_to_secondary`. If we find something nicer than the standard terminology we'll rename things.